README.md

```diff
@@ -0,0 +1,77 @@
+
+## How to Apply[^1]
+
+* Connect to the community
+* Get accustomed to the Unikraft development ecosystem
+* Make a small contribution
+* Decide on a GSoC project
+* Fill out an application
+
+### Connect to the community
+
+First join our [Discord server](https://bit.ly/UnikraftDiscord).
+This is where all (text) discussion and (live video) meetings take place.
+You will find multiple channels, each for a particular topic of the Unikraft ecosystem.
+
+As a newcomer, start by presenting yourself (name, university / occupation, interest in Unikraft, current experience).
+It's best if you used a `Firstname Lastname` ID to make it easy for others to get in touch with you.
+
+Ask any questions, worry not about anything, we're happy to assist you in feeling at home in the Unikraft community.
+In particular, use the `gsoc22` and the `support` channels.
+
+### Get accustomed to the Unikraft development ecosystem
+
+Unikraft development happens on the [Unikaft GitHub organization](https://github.com/unikraft/).
+
+For starts, check the first 2 sessions in [Unikraft Summer of Code](https://usoc21.unikraft.org/docs/).
+These give you a very nice set of initial hands-on tasks to get you started with grabbing, configuring, building and running Unikraft.
+Optionally, look into sessions 3, 4 and 5 as well.
+
+### Make a small contribution
+
+The best way to prepare your application is to make small contributions.
+
+First off, browse the [Unikraft Contribution Guidelines](https://unikraft.org/docs/contributing/).
+This give you an overview of how we expect you to write your code, create commits and make contributions and GitHub pull requests (PRs).
+
+Then aim for small, easy to to contributions.
+Look for items such as:
+* Removing build warnings for any of the repositories in the [Unikraft GitHub organization](https://github.com/unikraft/).
+* Fixing inconsistencies or adding missing information in the [documentation](https://github.com/unikraft/docs) (rendered on the [main Unikraft website](https://unikraft.org/)).
+* Adding [tests](https://unikraft.org/docs/develop/writing-tests/) to Unikraft components.
+
+### Decide on a GSoC project
+
+We put together a [list of project ideas](https://github.com/cs-pub-ro/gsoc22/blob/staging/content/en/ideas.md) that you can choose from.
+
+Note that this is not an exclusive list.
+You can suggest to new project ideas to the Unikraft GSoC mentors on the `gsoc22` channel on [Discord](https://bit.ly/UnikraftDiscord).
+If your idea has been vetted by a mentor (or more) in the community, then you can submit it as part of your application.
+
+If a given project idea tickles your fancy, ask about that on the `gsoc22` channel on [Discord](https://bit.ly/UnikraftDiscord).
```

Comment:
```suggestion
If you fancy a given project idea, ask about that on the `gsoc22` channel on [Discord](https://bit.ly/UnikraftDiscord).
```

I'm not sure about this expression 🤣. It might not be "appropriate" for the website. I'm fine with it either way.